Initialize auro-nav component (#4)
AlaskaAirlines/auro-nav: docs/partials/apiExamples.md

## API Examples

### Attribute Examples

#### Type
The `type` attribute determines the type of divider to be used to seperate all of the `auro-hyperlinks`.

##### Breadcrumb

> The default example when the `type=breadcrumb` attribute is present.

Reviewer: ```suggestion
Example when `auro-nav` slot is populated with `auro-nav-breadcrumb` elements.
```

Let's also add language explaining that the last breadcrumb should be the current page and that they should *not* populate the HREF attribute.

<div class="exampleWrapper">

>   <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/basic.html) -->

Reviewer: ```suggestion
  <!-- AURO-GENERATED-CONTENT:START (FILE:src=./../../apiExamples/breadcrumb.html) -->
```

Let's rename the example file(s) for this one.
